bitnami/grafana-loki/templates/NOTES.txt

```diff
@@ -0,0 +1,102 @@
+CHART NAME: {{ .Chart.Name }}
+CHART VERSION: {{ .Chart.Version }}
+APP VERSION: {{ .Chart.AppVersion }}
+
+** Please be patient while the chart is being deployed **
+
+{{- if .Values.diagnosticMode.enabled }}
+The chart has been deployed in diagnostic mode. All probes have been disabled and the command has been overwritten with:
+
+  command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 4 }}
+  args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 4 }}
+
+Get the list of pods by executing:
+
+  kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
+
+Access the pod you want to debug by executing
+
+  kubectl exec --namespace {{ .Release.Namespace }} -ti <NAME OF THE POD> -- bash
+
+In order to replicate the container startup execute this command:
+
+    loki -config.file=/bitnami/grafana-loki/conf/loki.yaml
+
+{{- else }}
+
+Installed components:
+
+  * ingester
+  * distributor
+  * querier
+  * query-frontend
+  * promtail
+  {{- if .Values.compactor.enabled }}
+  * compactor
+  {{- end }}
+  {{- if .Values.indexGateway.enabled }}
+  * index-gateway
+  {{- end }}
+  {{- if .Values.gateway.enabled }}
+  * gateway
+  {{- end }}
+  {{- if .Values.ruler.enabled }}
+  * ruler
+  {{- end }}
+  {{- if .Values.tableManager.enabled }}
+  * table-manager
+  {{- end }}
+
+{{- if .Values.gateway.enabled }}
+{{- if .Values.gateway.ingress.enabled }}
+
+1. Get the gateway URL and associate WordPress hostname to your cluster external IP:
```

Comment:
Same here, I think the text needs to change to `Grafana Loki`.

Reply:
Good catch! We are using the name of `gateway` as some lines above we are specifying the list of Loki components installed